projects/framework-blender/extensions/plugins/blender_scene_saved_validator.py

```diff
@@ -0,0 +1,82 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import bpy
+
+from ftrack_utils.paths import get_temp_path
+
+from ftrack_framework_core.plugin import BasePlugin
+from ftrack_framework_core.exceptions.plugin import (
+    PluginExecutionError,
+    PluginValidationError,
+)
+
+
+class BlenderSceneSavedValidatorPlugin(BasePlugin):
+    name = 'blender_scene_saved_validator'
+
+    def save_to_temp(self, store, extension_format):
+        '''
+        Save the file to a temporary location.
+        '''
+
+        try:
+            # Save file to a temp file
+            save_path = get_temp_path(filename_extension='.blend')
```

Comment:
You can use the extension format if you support more than one format, if not, remove the unussed argument.